src/store.ts

```diff
@@ -95,37 +95,42 @@ export function useStore(selector, options: SelectorOptions = {}): any {
   if (!store.updateFunctions[component.__owl__.id]) {
```

Comment:
maybe you could assign componentId above this line, it is used 3 times now...

Reply:
I was thinking that too but I already pushed the commit... I'll change it then.

Reply:
Rebased, added componentId, and replaced all the occurrence in the function